### PR TITLE
change display to load urdf content from parameter

### PR DESCRIPTION
**Pull request by Karsten1987**
connects to ros2/robot_state_publisher#15

This is work in progress. The PR so far introduces a new option for the robot model to load from parameter.
I haven't figured out yet though how to spawn a remote parameter client while the node is already up and running (inside an executor).

### CODE REVIEW

File: rviz_default_plugins/src/rviz_default_plugins/displays/robot_model/robot_model_display.cpp [hunk 1]
```diff
@@ -212,6 +217,22 @@ void RobotModelDisplay::load_urdf()
   }
 }
 
+void RobotModelDisplay::load_urdf_parameter(const std::string & parameter_name)
+{
+    printf("Loading urdf from parameter %s\n", parameter_name.c_str());
+    try {
+      auto param_client = rclcpp::SyncParametersClient(
```
Comment (Karsten1987) [inline rviz_default_plugins/src/rviz_default_plugins/displays/robot_model/robot_model_display.cpp:224]: this call does currently not work. It terminates with
```
Loading urdf from parameter robot_description
libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: Node has already been added to an executor.
Abort trap: 6
```

**Comment by jacobperron** [comment]
@Karsten1987 What's the status on this? I realize it is quite old, maybe it can just be closed.

**Comment by Karsten1987** [comment]
I would believe that it's still a valid use case. We could maybe backlog it?

**Comment by jacobperron** [comment]
> I would believe that it's still a valid use case. We could maybe backlog it?

Might be better to create an issue if you want to put in on the backlog. Is there anything blocking this or does it just need some TLC?

**Comment by clalancette** [comment]
I'm going to close this very old pull request, since we typically get the URDF from the published data from `robot_state_publisher`, rather than a parameter hosted here.  If this is really functionality we want in the future, we'll have to redo this and think about it anyway.

**Pull request closed without merging**